welcome/learn_more_about_openshift.adoc

```diff
@@ -303,20 +303,40 @@ a|* xref:../nodes/scheduling/nodes-scheduler-taints-tolerations.adoc#nodes-sched
 * xref:../machine_management/creating-infrastructure-machinesets.adoc#creating-infrastructure-machinesets[Creating infrastructure machine sets]
 |===
 
+[id="self-managed-hcp"]
+== {hcp-capital}
 
-//
-//[id="hosted-control-plane-activities"]
-//== Hosted control plane activities
+[options="header",cols="2*"]
+|===
+|Learn about {hcp} |Optional additional resources
 
-//[options="header",cols="2*"]
-// |===
-// |Learn about {hcp-capital} |Optional additional resources
+| xref:../hosted_control_planes/index.adoc#hosted-control-planes-overview[Hosted control planes overview]
+a|
+xref:../hosted_control_planes/index.adoc#hosted-control-planes-version-support_hcp-overview[Versioning for {hcp}]
 
-// | xref:../hosted_control_planes/index.adoc#hosted_control_planes[Hosted control planes overview]
-// | xref:../hosted_control_planes/index.adoc#hosted-control-planes-overview_hcp-overview[Introduction to hosted control planes]
+| Preparing to deploy 
+a| * xref:../hosted_control_planes/hcp-prepare/hcp-requirements.adoc#hcp-requirements[Requirements for {hcp}]
+* xref:../hosted_control_planes/hcp-prepare/hcp-sizing-guidance.adoc#hcp-sizing-guidance[Sizing guidance for {hcp}]
+* xref:../hosted_control_planes/hcp-prepare/hcp-override-resource-util.adoc#hcp-override-resource-util[Overriding resource utilization measurements]
+* xref:../hosted_control_planes/hcp-prepare/hcp-cli.adoc#hcp-cli[Installing the {hcp} command-line interface]
+* xref:../hosted_control_planes/hcp-prepare/hcp-distribute-workloads.adoc#hcp-distribute-workloads[Distributing hosted cluster workloads]
+* xref:../hosted_control_planes/hcp-prepare/hcp-enable-disable.adoc#hcp-enable-disable[Enabling or disabling the {hcp} feature]
 
-// | xref:../hosted_control_planes/hcp-getting-started.adoc#hcp-getting-started [Getting started with hosted control planes]
-// |
+| Deploying {hcp}
+a| * xref:../hosted_control_planes/hcp-deploy/hcp-deploy-virt.adoc#hcp-deploy-virt[Deploying {hcp} on {VirtProductName}] 
+* xref:../hosted_control_planes/hcp-deploy/hcp-deploy-aws.adoc#hcp-deploy-aws[Deploying {hcp} on {aws-short}] 
+* xref:../hosted_control_planes/hcp-deploy/hcp-deploy-bm.adoc#hcp-deploy-bm[Deploying {hcp} on bare metal] 
+* xref:../hosted_control_planes/hcp-deploy/hcp-deploy-non-bm.adoc#hcp-deploy-non-bm[Deploying {hcp} on non-bare-metal agent machines] 
+* xref:../hosted_control_planes/hcp-deploy/hcp-deploy-ibmz.adoc#hcp-deploy-ibmz[Deploying {hcp} on {ibm-z-title}] 
+* xref:../hosted_control_planes/hcp-deploy/hcp-deploy-ibmpower.adoc#hcp-deploy-ibmpower[Deploying {hcp} on {ibm-power-title}]
+
+| xref:../hosted_control_planes/hcp-disconnected/hcp-deploy-dc.adoc#hcp-deploy-dc[Deploying {hcp} in a disconnected environment]
```

Comment:
It's a bit confusing that this link actually goes to a topic called _Introduction to hosted control planes in a disconnected environment_. (That assembly does seem out of place; not sure where one would put it.)

Reply:
Good catch. I think I'll just remove the link from the phrase "Deploying hosted control planes in a disconnected environment".